### PR TITLE
Rev-006-create-signup-page

### PR DESCRIPTION
**Pull request by IbrahimAldereni-socuimtech**
SIGNUP-PAGE

- figma link:
https://www.figma.com/file/22d3bKcFL0ipWIZJOUY3Km/reverifi?node-id=0%3A14569

### CODE REVIEW

File: src/pages/SignUp/SignUp.jsx [hunk 1]
```diff
@@ -0,0 +1,178 @@
...
+    phonePrefix: yup
+      .number()
+      .positive()
+      .integer()
+      .when('phoneCheckbox', {
+        is: true,
+        then: yup.number().required().typeError('phonePrefix must be a number'),
+      }),
+  })
+  .required();
```
Comment (NoorAbuzainahOP) [inline src/pages/SignUp/SignUp.jsx:59]: can we create a separate file for the schema? perhaps src/pages/SignUp/SignUpSchema.js?

File: src/pages/SignUp/SignUp.jsx [hunk 2]
```diff
@@ -0,0 +1,178 @@
...
+    register,
+    handleSubmit,
+    formState: { errors },
+    reset,
+  } = useForm({
+    resolver: yupResolver(schema),
+  });
+
+  const onSubmit = (data) => {
+    console.log('form data', data);
```
Comment (NoorAbuzainahOP) [inline src/pages/SignUp/SignUp.jsx:74]: do we need this here?

**Comment by NoorAbuzainahOP** [comment]
For some reason, the style of the background image changes according to the 'I am an industry professional' check value, is that normal ?

Screen Shot 2021-12-23 at 6 54 28 PM
Screen Shot 2021-12-23 at 6 54 51 PM

File: src/pages/Login/Login.test.jsx [hunk 3]
```diff
@@ -0,0 +1,6 @@
+describe('test2', () => {
```
Comment (Mahmoudzaid2994) [inline src/pages/Login/Login.test.jsx:1]: what is this??

File: src/pages/SignUp/SignUp.test.jsx [hunk 4]
```diff
@@ -0,0 +1,6 @@
+describe('test', () => {
```
Comment (Mahmoudzaid2994) [inline src/pages/SignUp/SignUp.test.jsx:1]: also here??

File: src/pages/SignUp/SignUp.jsx [hunk 5]
```diff
@@ -0,0 +1,147 @@
...
+            Continue with Facebook
+          </SocialButton>
+          <SocialButton dark>
+            <AppleIcon />
+            Continue with Apple
+          </SocialButton>
+          <SocialButton light>
+            <GoogleIcon />
+            Continue with Google
+          </SocialButton>
```
Comment (NoorAbuzainahOP) [inline src/pages/SignUp/SignUp.jsx:136]: suggestion: can we try to use a map for this?
```  
const buttons = [
    { color: { blue: true }, icon: <FacebookIcon />, name: 'Facebook' },
    { color: { dark: true }, icon: <AppleIcon />, name: 'Apple' },
    { color: { light: true }, icon: <GoogleIcon />, name: 'Google' },
  ];

```

**Review by NoorAbuzainahOP — APPROVED** [review]
I left a minor suggestion, other than LGTM

**Pull request merged**